bot/exts/filtering/filtering.py

```diff
@@ -1432,7 +1434,17 @@ async def send_weekly_auto_infraction_report(self, channel: discord.TextChannel
         if len(lines) == 1:
             lines.append("Nothing to show")
 
-        await channel.send("\n\n".join(lines))
+        report = "\n\n".join(lines)
+        if len(report) > MAX_PASTE_LENGTH:
```

Comment:
`MAX_PASTE_LENGTH` is the maximum length that can be sent to the pastebin which isn't what we want here. Not sure if we have a constant anywhere for message length.

Reply:
🤦 I'll sort it